modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -158,22 +166,30 @@ namespace UDIMPL {
     return distribution(gen);
   }
 
-  // other UDFs ...
-
-}
-
-
 #endif /* EXPRFUNCTIONS_HPP_ */
 ----
 
+After adding your query, use the `PUT` command to upload the file. This will upload the file to all nodes in a cluster:
 
-
-Lastly, use the `PUT` command to upload the file. This will uploaded the file to all nodes in a cluster:
-
-[source,text]
+[source,gsql]
 ----
-GSQL > PUT ExprFunction FROM "/home/tigergraph/documents/udf.hpp"
+GSQL > PUT ExprFunction FROM "/path/to/udf.hpp"
 PUT ExprFunctions successfully.
 ----
 
-The UDF has now been added to GSQL and you can start using the function in GSQL queries.
+The UDF has now been added to GSQL. You can `INSTALL` and `RUN` the function in GSQL queries.
+
```

Comment:
1. A UDF is different from a query. `INSTALL QUERY` and `RUN QUERY` work on queries only. 
2. Don't use code words as English words.